README.md

```diff
@@ -30,7 +30,7 @@ If Windows SmartScreen prevents you from running the program click `More info` a
 ## On Linux: 
 Run `./start.sh` (or `bash start.sh`) in a terminal.
 
-The installer will take care of whatever is needed. A friendly [Discord community](https://discord.com/invite/u9yhsFmEkB) will help you if you face any problems.
+The installer will take care of whatever is needed. If you face any problems, join the [Discord](https://discord.com/invite/u9yhsFmEkB) and ask fo assistance.
```

Comment:
1. `fo` -> `for`
2. I'd like to keep the "friendly" adjective before `Discord`, and the word "community", so `joint the friendly [Discord community] and ask for assistance..`.

Reply:
Changed to: